alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/AlertService.kt

```diff
@@ -153,6 +161,55 @@ class AlertService(
         }
     }
 
+    // TODO: Change the parameters to use result: AggTriggerRunResult instead of currentAlerts and aggResultBuckets after integration
+    fun getCategorizedAlertsForAggregationMonitor(
```

Comment:
nit: This function can be refactored by using Set instead of List and defining the right comparator in Alert class. And a new function maybe - `getAlertFromAggResultBucket()`. Changing the content of completed alert object towards the end might still be required.

Reply:
Sure, we can have a follow-up task to revisit this and clean it up.